### ISSUE TITLE
Release for PyPI

### ISSUE THREAD

**Issue by vnitinv** [robshakir/pyangbind#20]
If we release the module to PyPI we wont face below issues while importing
```python
>>> import bgp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "bgp/__init__.py", line 3, in <module>
    from lib.yangtypes import RestrictedPrecisionDecimalType, RestrictedClassType, TypedListType
ImportError: No module named lib.yangtypes
```
currently to get rid of this issue, we have to define PYTHONPATH to the cloned folder path.

**Comment by robshakir** [comment]
Hi Nitin,

Sure -- we can look at that. At the moment rather than having to define PYTHONPATH everywhere, you can use the --pybind-class-dir switch to pyang, and that will append the path specified to the Python that using the `sys` module:

```
import sys
sys.path.append("/Users/rjs/Code/pyangbind")
```

It'd definitely be better to have this in an importable module - it is just a case of ensuring that the module is stable enough.

Cheers,
r.

**Comment by robshakir** [comment]
I did the work to package for PyPI -- please try the test package at https://testpypi.python.org/pypi/PyangBind/0.1.1 -- and check the packaging branch.

**Comment by robshakir** [comment]
Closed by #26
